subsys/bluetooth/host/Kconfig.l2cap

```diff
@@ -33,6 +33,7 @@ config BT_L2CAP_TX_MTU
 	default 69 if BT_MESH_GATT
 	default 65 if BT_SMP
 	default 64 if BT_AUDIO_UNICAST_SERVER || BT_AUDIO_UNICAST_CLIENT || BT_AUDIO_BROADCAST_SINK || BT_BASS || BT_BASS_CLIENT
+	default 49 if BT_HAS || BT_HAS_CLIENT
```

Comment:
The non-SC SMP can fit in 23 bytes of MTU. The 65 bytes are required by SC (to send Public Keys). I believe this somehow can be tweaked someday, so that the BT_SMP default MTU value could be 23 is SC will not be enabled. The clue is that it is safe to keep the MTU required by the HAS specification, as the OOB can be used (the SC is not mandatory, but optional)

Reply:
Right - Thanks for reminding me :) Currently when BT_HAS depends on SMP it won't really do much, but as you mentioned last time the MTU for BT_SMP may be changed.